Gemfile.lock

```diff
@@ -541,8 +541,8 @@ DEPENDENCIES
   pry-stack_explorer
   quiet_assets
   rabl (= 0.9.3)
-  rack-attack
   rack (~> 1.4.7)
+  rack-attack
```

Comment:
Arg, yes I did... for some reason my bundler will always switch the position of `rack-attack` in the `Gemfile.lock`.

Now that it slipped through I would ask to try and keep the change in, in the hope that no other bundler will want to switch it back again...

FWIW, I am using:
> Bundler version 1.10.5